### ISSUE TITLE
able to add blank values in keywords

### ISSUE THREAD

**Issue by RussellSPG** [samtecspg/articulate#849]
### Issues
> *If you're having trouble with Articulate we definitely want to help you. If you can provide a little bit of information it will make our troubleshooting job easier.*

Noticed this, running some functionality re-testing, that blank keywords can be added. 
Please describe the issue you are having here and answer the question below.

![image](https://user-images.githubusercontent.com/42249324/62735311-e7976f00-b9f8-11e9-9947-cf2ebaf384f0.png)

Also, this error message as a place holder could perhaps be me clear to just say "can't train because keyword value is blank IMO.
```
Error: child "examples" fails because ["examples" at position 2 fails because [child "value" fails because ["value" is not allowed to be empty]]]. 😖
```

0.27.2
win 10 pro
FF
no errors
--

**Comment by wrathagom** [comment]
@dcalvom can you confirm this and get it patched? No reason to interrupt the button work, just whenever you get to it.

**Comment by RussellSPG** [comment]
Edit: Issue was also present in 27.1

**Comment by dakavisa** [comment]
Closed by 21abf670ea29f0ae19a3fca20beddd97b72229fd
